src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const whoIsPartner = sex === 'f'
+    ? `${partnerName} is my husband`
+    : `${partnerName} is my wife`;
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+        {age
+          && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+          )}
+
+        <p className="Person__partner">
+          {isMarried ? `${whoIsPartner}` : 'I am not married'}
```

Comment:
```suggestion
          {isMarried 
            ? `${whoIsPartner}` 
            : 'I am not married'
          }
```
![image](https://github.com/mate-academy/react_person/assets/96409563/b55aa8fd-d336-4678-8e33-c5df309c29c3)